lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -275,7 +275,9 @@ RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme.conf .
 RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme_fd.conf .
 
 # Create running dirs
-WORKDIR /var/opt/magma
+WORKDIR /var/opt/magma/configs
```

Comment:
Here you use /var/opt/magma/configs and on line /var/opt/magma/config
Which is right?

Reply:
Hi @mattymo 

I just followed what @ssanadhya told me  in a [slack message](https://magmacore.slack.com/archives/C01B32Q9L76/p1612977897017200)

Reply:
Good catch @mattymo ! the override configs should always be in `/var/opt/magma/configs`

Reply:
Fixed the typo! Thx @mattymo and @ssanadhya